universal-application-tool-0.0.1/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java

```diff
@@ -57,14 +59,13 @@ protected ReadOnlyApplicantProgramServiceImpl(
 
   @Override
   public Optional<Block> getBlockAfter(Block block) {
-    return getBlockAfter(block.getId());
+    return getBlockAfter(block.id());
   }
 
   // TODO(https://github.com/seattle-uat/universal-application-tool/issues/224): Implement checking
   // blocks for completion
```

Comment:
Suggestion (blocking): please remove the comment if it is done.

Reply:
Thanks for catching that!